tests/format/js/if/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -93,26 +93,44 @@ parsers: ["babel", "flow", "typescript"]
 printWidth: 80
                                                                                 | printWidth
 =====================================input======================================
-if (a === 0) doSomething(); // comment A1
+     if (a === 0) doSomething(); // comment A1
 else if (a === 1) doSomethingElse(); // comment B1
 else if (a === 2) doSomethingElse(); // comment C1
 
-if (a === 0) doSomething(); /* comment A2 */
+     if (a === 0) doSomething(); /* comment A2 */
 else if (a === 1) doSomethingElse(); /* comment B2 */
 else if (a === 2) doSomethingElse(); /* comment C2 */
 
-if (a === 0) expr; // comment A3
+     if (a === 0) expr; // comment A3
 else if (a === 1) expr; // comment B3
 else if (a === 2) expr; // comment C3
 
-if (a === 0) expr; /* comment A4 */
+     if (a === 0) expr; /* comment A4 */
 else if (a === 1) expr; /* comment B4 */
 else if (a === 2) expr; /* comment C4 */
 
-if (a === 0) looooooooooooooooooooooooooooooooooooooooooooooooooooooooooooong; // comment A5
+     if (a === 0) looooooooooooooooooooooooooooooooooooooooooooooooooooooooooooong; // comment A5
 else if (a === 1) looooooooooooooooooooooooooooooooooooooooooooooooooooooooooooong; // comment B5
 else if (a === 2) looooooooooooooooooooooooooooooooooooooooooooooooooooooooooooong; // comment C5
 
+function a() {
+  if (a) return; /* comment 6a */
+  else return 2;
+
+  if (a) return 1; /* comment 6b */
+  else return 2;
+
+  if (a) throw e; /* comment 6d */
+  else return 2;
+
+  // TODO[@fisker]: fix this
+  // if (a) var a = 1; /* comment 6e */
+  // else return 2;
```

Comment:
I forget... but I tried to run tests in local with:

```diff
-        addDanglingComment(
-          precedingNode,
-          comment,
-          markerForIfWithoutBlockAndSameLineComment,
-        );
+        addTrailingComment(precedingNode, comment);
```

then I got the spanpshot diffs like this:

```diff
- if (a === 0) expr; // comment A3
- else if (a === 1) expr; // comment B3
+ if (a === 0)
+   expr; // comment A3
+ else if (a === 1)
+   expr; // comment B3
```

At the time, I may have wanted to avoid this line break.